tests/Doctrine/Tests/ORM/Functional/Ticket/GH9376Test.php

```diff
@@ -0,0 +1,133 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Tests\ORM\Functional\Ticket;
+
+use Doctrine\ORM\Mapping as ORM;
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Entity;
+use Doctrine\ORM\Mapping\GeneratedValue;
+use Doctrine\ORM\Mapping\Id;
+use Doctrine\Tests\OrmFunctionalTestCase;
+
+class GH9376Test extends OrmFunctionalTestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        $this->_schemaTool->createSchema([
+            $this->_em->getClassMetadata(GH9376GiftVariant::class),
+            $this->_em->getClassMetadata(GH9376Product::class),
+            $this->_em->getClassMetadata(GH9376Gift::class),
+        ]);
+    }
+
+    protected function tearDown(): void
+    {
+        $this->_schemaTool->dropSchema([
+            $this->_em->getClassMetadata(GH9376GiftVariant::class),
+            $this->_em->getClassMetadata(GH9376Product::class),
+            $this->_em->getClassMetadata(GH9376Gift::class),
+        ]);
+
+        parent::tearDown();
+    }
```

Comment:
Please remove `tearDown()` and use `createSchemaForModels()`. See https://github.com/doctrine/orm/pull/9526 for more details about the why.